src/components/Card/card.css

```diff
@@ -0,0 +1,49 @@
+
+@media screen and (max-width: 600px) {
+  /* Mobile View */
+  .card {
+    max-width: 80%;
+    min-width: 80%;
+    height: 400px;
+    background-color: #355878;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.5), 0 6px 20px 0 rgba(0, 0, 0, 0.4);
+    color: #fff;
+    margin-top: 30px;
+  }
```

Comment:
we should make the whole card component clickable to the single mentor page.  Would be good to add the cursor pointer.
```cursor: pointer;```